velox/duckdb/conversion/DuckWrapper.cpp

```diff
@@ -201,9 +227,7 @@ VectorPtr convert(
       // Unused dictionary elements can be uninitialized. That can cause
       // errors if we try to decode them. Here we create a bitmap of
       // used values to avoid that.
-      if (child.GetType() == LogicalTypeId::HUGEINT ||
-          child.GetType() == LogicalTypeId::TIMESTAMP ||
-          child.GetType() == LogicalTypeId::VARCHAR) {
+      if (hasValidityVector(child.GetType().id())) {
```

Comment:
hasValidityVector name is confusing. All DuckDB vectors have a validity vector, I think.

Reply:
I am removing this method for now.